core/string/ustring.cpp

```diff
@@ -1626,11 +1626,6 @@ String String::to_lower() const {
 	return lower;
 }
 
-String String::chr(char32_t p_char) {
-	char32_t c[2] = { p_char, 0 };
-	return String(c);
```

Comment:
This seems to be equivalent to
```
string.parse_utf32(Span<char32_t>(&p_char, 1));
```
rather than
```
string.parse_utf32(p_char);
```

Reply:
Not exactly. The old implementation would have converted a `NUL` char to a zero-size string, while the new one will complain about it. This is also true for the `string.parse_utf32(Span<char32_t>(&p_char, 1));` version.